packages/dso-toolkit/components/Voorbeelden/30-Componenten/application-headings.config.yml

```diff
@@ -0,0 +1,26 @@
+status: ready
+label: Application Heading
+context:
+  __title: default
+  title: Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
+variants:
+  - name: with-substitel
+    context:
+      __title: Met subtitel
+      subtitle: Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
+  - name: with-subtitel-and-steps
+    context:
+      __title: Met subtitel en stappen
+      subtitle: Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
+      step: Stap x van x
+  - name: only-substitel
```

Comment:
only subs? of only sub?

Reply:
ik heb gekozen voor `just-subtitle` en daaronder `just-subtitle-and-steps`